src/main/java/org/kainos/ea/db/JobspecDAO.java

```diff
@@ -0,0 +1,34 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.Job;
+
+import java.sql.*;
```

Comment:
IntelliJ will have done this to you, but can you make this not be a wildcard import?